tutorials/introductory/animation_tutorial.py

```diff
@@ -0,0 +1,274 @@
+"""
+===========================
+Animations using matplotlib
+===========================
+
+Based on its plotting functionality, matplotlib also provides an interface to
+generate animations using the :class:`~matplotlib.animation` module. An
```

Comment:
Just for your info: we have setup sphinx to be very "forgiving" when it comes to providing full paths etc. Here, I am quite sure that 
```
:class:`~matplotlib.animation`
```
can be changed to
```
`.animation`
``` 
and so on. It may often be enough to just have 
```
.name
```
instead of
```
~matplotlib.module.name
```
(the build will complain if there is more than one option, typically for methods/functions that are both in Axes and pyplot, and Axes itself).

Will save you a bit of writing if anyhing else.

Reply:
Understood. Thanks!